assets/src/edit-story/elements/text/display.js

```diff
@@ -88,7 +92,22 @@ function TextDisplay({
     maybeEnqueueFontStyle(fontFamily);
   }, [fontFamily, maybeEnqueueFontStyle]);
 
-  return <Element dangerouslySetInnerHTML={{ __html: content }} {...props} />;
+  useTransformHandler(id, (transform) => {
+    const target = ref.current;
+    const updatedFontSize =
+      transform && transform.updates && transform.updates.fontSize;
```

Comment:
Can use optional chaining here,  i.e. `const updatedFontSize = transform?.updates?.fontSize`

Reply:
Nice!

Reply:
I'm wondering if we there's a linter that could do that.